.travis.yml

```diff
@@ -2,7 +2,7 @@
 
 language: node_js
 node_js:
-  - "6.9"
```

Comment:
I kept getting an error during `npm install` when I used version 6.9 locally, but I was able to get things working using node 8.